test/resources/sampleContainer/example1.ttl

```diff
@@ -2,6 +2,8 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/> .
 @prefix ex: <http://example.org/stuff/1.0/> .
 
+</sampleContainer/example*> a :glob.
```

Comment:
FYI, this is not how you test glob. `</sampleContainer/example*> a :glob.` being in a document makes no sense to me.